modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -25,75 +25,79 @@ $ oc get all,ep,cm -n openshift-ovn-kubernetes
-configmap/ovnkube-config             1      57m
-configmap/signer-ca                  1      57m
+configmap/control-plane-status       1      113m
+configmap/kube-root-ca.crt           1      114m
+configmap/openshift-service-ca.crt   1      114m
+configmap/ovn-ca                     1      114m
+configmap/ovnkube-config             1      114m
+configmap/signer-ca                  1      114m
+
 ----
 +
-There are three `ovnkube-masters` that run on the control plane nodes, and two daemon sets used to deploy the `ovnkube-master` and `ovnkube-node` pods.
-There is one `ovnkube-node` pod for each node in the cluster.
-In this example, there are 5, and since there is one `ovnkube-node` per node in the cluster, there are five nodes in the cluster.
-The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations started by online-master and `ovnkube-node`.
-The `ovn-kubernetes-master` `ConfigMap` has the information of the current online master leader.
+There are three `ovnkube-cluster-manager` that run on the control plane nodes, and one daemon set used to deploy the `ovnkube-control-plane` and `ovnkube-node` pods.
+There is one ovnkube-node pod for each node in the cluster.
+The `ovnkube-config` `ConfigMap` has the {product-title} OVN-Kubernetes configurations.
++
+[NOTE]
+====
+For Azure, there will be eleven `ovnkube-node` pods in a cluster.
+====
 
-. List all the containers in the `ovnkube-master` pods by running the following command:
+. List all of the containers in the `ovnkube-node` pods by running the following command:
 +
 [source,terminal]
 ----
-$ oc get pods ovnkube-master-9g7zt \
--o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
+$ oc get pods ovnkube-node-bcvts -o jsonpath='{.spec.containers[*].name}' -n openshift-ovn-kubernetes
 ----
 .Expected output
 +
 [source,terminal]
 ----
-northd nbdb kube-rbac-proxy sbdb ovnkube-master ovn-dbchecker
+ovn-controller ovn-acl-logging kube-rbac-proxy-node kube-rbac-proxy-ovn-metrics northd nbdb sbdb ovnkube-controller
 ----
-+
-The `ovnkube-master` pod is made up of several containers.
-It is responsible for hosting the northbound database (`nbdb` container), the southbound database (`sbdb` container), watching for cluster events for pods, egressIP, namespaces, services, endpoints, egress firewall, and network policy and writing them to the northbound database (`ovnkube-master` pod), as well as managing pod subnet allocation to nodes.
+The `ovnkube-node` pod is made up of several containers. It is responsible for hosting the northbound database (`nbdb` container), the southbound database (`sbdb` container), the north database (`northb` container), and the `ovn-controller` container. The `ovn-controller` container watches for cluster events for pods, egressIP, namespaces, services, endpoints, egress firewall, and network policy, and allocates pod IP from pod subnet to each pod through IP address management (IPAM).
```

Comment:
```suggestion
The `ovnkube-node` pod is made up of several containers. It is responsible for hosting the northbound database (`nbdb` container), the southbound database (`sbdb` container), the north daemon (`northd` container), `ovn-controller` and the `ovnkube-controller`container. The `ovnkube-controller` container watches for API objects like pods, egressIPs, namespaces, services, endpoints, egress firewall, network policy etc. It is also responsible for allocating pod IP from the available subnet pool for that node.
```

Reply:
I see what you mean now about `ovnkube-controller`.